packages/playground/src/shared/dom/Scene.tsx

```diff
@@ -76,8 +76,18 @@ const Box: React.FC<{
   sheet: ISheet
   selection: IStudio['selection']
 }> = ({id, sheet, selection}) => {
+  const defaultConfig = useMemo(
+    () =>
+      Object.assign({}, boxObjectConfig, {
+        // give the box initial values offset from each other
+        x: ((id.codePointAt(0) ?? 0) % 15) * 100,
```

Comment:
I understand this results in offset boxes, not sure what this is doing though. Is it like a random number generator? why `id.codePointAt(0)`? 

Oh I think I see, the ids are numbers that are increasing so this works like that? 

Break out into function?

Reply:
I don't really care enough to try to pull it out. Essentially, it just uses the first char of the string as a reference of code point (e.g. `"A".charCodeAt(0)` -> `65`) then uses that as a modulo'd number with 100 px offset. It's dirty, but I don't care too much since the comment is there. To break it out puts a special kind of pressure on the code being clean / re-usable which it absolutely isn't 😂

Reply:
fair enough